website/docs/reference/notifications.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Notifications
+---
+
+:::info Availability
+
+Notifications were introduced in Unleash 4.22.0 for pro and enteprise customers.
+
+:::
+
+Unleash's notifications give you updates when certain events occur in projects that you are part of. The notifications are accessible from the notifications button in the navigation section of the admin UI.
+
+Your notifications will only contain updates from other members; you will not get notifications about actions you perform yourself.
+
+![The notifications overview. It's telling the user about a feature that was enabled in production and that there is a change request ready for review.](/img/notifications.png)
+
+From within the notifications list, you can:
+* navigate to the features or change request that the notification is for
+* mark all notifications as read
+* filter out unread notifications
+Notifications can not be disabled.
```

Comment:
Last thing: you'll need a space here for it not to be part of the last list item.

```suggestion

Notifications can not be disabled.
```